Makefile

```diff
@@ -38,6 +39,7 @@ ifeq ($(UNAME_S),Darwin)
 	endif
 endif
 ifeq ($(UNAME_S),FreeBSD)
+	MAKE ?= gmake
```

Comment:
Should this be `MAKE = gmake`? Otherwise `MAKE` will already always be set by the first line.

Reply:
Whoops, screwed up when pulling these changes into git...

Reply:
Does BSD make (e.g., `bmake`) understand `ifeq` syntax? If not whatever invokes `make` needs to be changed. `MAKE` variable in makefiles is [implicitly defined](https://www.gnu.org/software/make/manual/html_node/MAKE-Variable.html) by make implementation.